src/reader.lisp

```diff
@@ -240,34 +240,37 @@ Used to forbid reading while inside quasiquoted forms.")
                      (let ((*coalton-reader-allowed* t))
                        (funcall (get-macro-character #\, (named-readtables:ensure-readtable :standard)) s c)))))
 
+(defun print-form (form)
+  "Prevent truncation of a FORM that will be immediately re-read."
+  (let ((*print-length* nil)
+        (*print-level* nil)
+        (*print-circle* t))
```

Comment:
I *think* setting `*default-float-format*` to double-float will fix #1073.

Reply:
It doesn't.